Source/WebCore/editing/FrameSelection.h

```diff
@@ -67,7 +67,7 @@ class CaretBase {
     void clearCaretRect();
     bool updateCaretRect(Document&, const VisiblePosition& caretPosition);
     bool shouldRepaintCaret(const RenderView*, bool isContentEditable) const;
-    void paintCaret(const Node&, GraphicsContext&, const LayoutPoint&, CaretAnimator* = nullptr) const;
+    void paintCaret(const Node&, GraphicsContext&, const LayoutPoint&, CaretAnimator*, const std::optional<VisibleSelection>&) const;
```

Comment:
You could keep the default args?
```suggestion
    void paintCaret(const Node&, GraphicsContext&, const LayoutPoint&, CaretAnimator* = nullptr, const std::optional<VisibleSelection>& = std::nullopt) const;
```